financialaid/templates/review_financial_aid.html

```diff
@@ -1,116 +1,409 @@
               </td>
+              {% endif %}
+              <!-- /Actions -->
+            </tr>
+            <!-- /Financial aid application information -->
+            <!-- Financial aid application email form -->
+            {% if selected_status == statuses.PENDING_MANUAL_APPROVAL or selected_status == statuses.PENDING_DOCS or selected_status == statuses.DOCS_SENT %}
+            <tr id="application-email-row-{{ financial_aid.id }}" style="display: none;">
+              <td colspan="9" style="border-top: 0;">
+                <form id="email-form-{{ financial_aid.id }}" class="email-form form-horizontal">
+                  {% render_form email_serializer template_pack="rest_framework/horizontal" %}
+                </form>
+                <button class="btn btn-primary pull-right"
+                        onclick="sendEmail({{ financial_aid.id }}, '{% url 'financial_aid_mail_api' financial_aid_id=financial_aid.id %}');">Send Email</button>
+                <br>
+              </td>
             </tr>
+            {% endif %}
+            <!-- /Financial aid application email form -->
           {% endfor %}
           </tbody>
         </table>
+        <!-- /Financial aid application table -->
       </div>
     </div>
   </div>
+  
   {% include "footer.html" %}
+  <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <script>
+    
+    var CSRF_TOKEN = "{{ csrf_token }}";
+    
+    {% if selected_status == statuses.PENDING_DOCS or selected_status == statuses.DOCS_SENT %}
+    /**
+     * Marks documents as received for a financial aid application
+     * 
+     * @param financial_aid_id {number} Financial aid application id
+     * @param url {string} URL to submit request to
+     */
+    function submitDocsReceived(financial_aid_id, url) {
+      var name = $("#full-name-" + financial_aid_id).text().trim();
+      if (confirm("Click OK to mark documents as received for " + name + "'s financial aid application.")) {
+        var action = "{{ statuses.PENDING_MANUAL_APPROVAL }}";
+        $.ajax({
+          "url": url,
+          "type": "PATCH",
+          "headers": {
+            "X-CSRFToken": CSRF_TOKEN
+          },
+          "data": {
+            "action": action
+          },
+          "success": function() {
+            displayMessage(
+              "Successfully marked documents as received for " + name + "'s financial aid application.",
+              "success"
+            );
+            $("#application-row-" + financial_aid_id + ", #application-email-row-" + financial_aid_id).remove();
+          },
+          "error": function(result) {
+            displayMessage("Error: " + result.responseText + " on " + name + "'s financial aid application.", "danger");
+          }
+        });
+      }
+    }
+    {% endif %}
+    
+    
+    {% if selected_status == statuses.PENDING_MANUAL_APPROVAL %}
+    /**
```

Comment:
Can you move all this JS to a separate file in `static/js`? We also don't need to define the functions conditionally, it would be better if they were always defined